apps/bfd-server/bfd-server-war/pom.xml

```diff
@@ -42,6 +42,12 @@
 	</dependencyManagement>
 
 	<dependencies>
+		<dependency>
+			<!-- BFD Shared Utilities include a wrapper around MDC for logging. -->
+			<groupId>gov.cms.bfd</groupId>
+			<artifactId>bfd-server-shared-utils</artifactId>
+			<version>${parent.version}</version>
```

Comment:
This will work but using project.version is a bit more consistent with the other projects.